app/screens/send-bitcoin-screen/send-bitcoin-screen.tsx

```diff
@@ -709,14 +719,12 @@ export const SendBitcoinScreenJSX: ScreenType = ({
           errorStyle={{ fontSize: 16, alignSelf: "center", height: 18 }}
           editable={false}
           selectTextOnFocus
-          InputComponent={(props) =>
-            fee === undefined ? (
-              <ActivityIndicator animating size="small" color={palette.orange} />
-            ) : fee === -1 ? (
-              <Text>{translate("SendBitcoinScreen.feeCalculationUnsuccessful")}</Text> // todo: same calculation as backend
-            ) : (
-              <TextInput {...props} />
-            )
+          InputComponent={
+            fee === undefined
+              ? FeeActivityIndicator
+              : fee === -1
+              ? FeeCalculationUnsuccessfulText
+              : TextInput
```

Comment:
This double ternary here is a bit hard to digest

Reply:
I only refactored the `feeInputComponent`, but I'll do the others over in another PR if you approve of the refactoring.